src/loader.coffee

```diff
@@ -8,6 +8,16 @@ library without blocking the `window.onload` event.
 @see http://goo.gl/VLDc3F More information on FiF technique
 ###
 ((url)->
+  try
+    # Avoid loading multiple Skroutz Analytics scripts, which leads to
+    # load Payload multiple times.
+    window["@@flavorAnalyticsLoader"] ||= { count: 0 }
+    window["@@flavorAnalyticsLoader"].count += 1
+
+    if window["@@flavorAnalyticsLoader"].count > 1
+      return console.warn("@@flavorAnalytics loaded #{window["@@flavorAnalyticsLoader"].count} times")
+  catch _err
```

Comment:
We can omit the error argument or even better the catch itself, since we don't use them.
https://coffeescript.org/#try

Reply:
👌